README.md

```diff
@@ -88,14 +88,30 @@ your Fluo repo or if you just want just want a fresh install.  To view your inst
 cd install/fluo-1.0.0-beta-1-SNAPSHOT
 ```
 
-From here you can run the `fluo` command to administer Fluo.
+Verify your installation by running the `fluo` command which you can use to administer Fluo.
 
 ```
 bin/fluo
 ```
 
-Next, follow the instructions starting at the [Configure a Fluo application][2] section of the 
-Fluo production setup instructions to configure, initialize, and start a Fluo application
+Running Fluo applications
+-------------------------
+
+There are two ways to run Fluo applications using `fluo-dev`:
+
+1. Run pre-built Fluo applications like [fluo-stress] that are listed and configured 
+   in `conf/applications.props` and can be run by a single command.
+
+    ```
+    fluo-dev run stress
+    ```
+
+   The `run` command will clone the application's repo to `install/fluo-app-repos/`.  It will
+   run the commands specified in `applications.props` to initialize and run the application
+   which will remain running unless you stop it using `fluo stop <app>`.
```

Comment:
Could add a sentence about arguments being passed through.   I saw you added something to the command help, if you think thats sufficient I am ok w/ that.